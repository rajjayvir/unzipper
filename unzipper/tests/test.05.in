0x09
0x67 0x10 0x00
0x66 0x28 0x00
0x65 0x3c 0x00
0x64 0x4e 0x00
0x63 0x5f 0x00
0x0a 0x6f 0x80
0x78 0x8f 0xc0
0x61 0x8f 0xd0
0x62 0x7f 0xe0
0x00 0x00 0x00 0x00
