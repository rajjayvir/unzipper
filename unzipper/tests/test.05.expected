0x67 : 1 0
0x66 : 2 10
0x65 : 3 110
0x64 : 4 1110
0x63 : 5 11110
0x0a : 6 111110
0x78 : 8 11111100
0x61 : 8 11111101
0x62 : 7 1111111
